src/sentry/models/project.py

```diff
@@ -249,12 +249,19 @@ def get_member_alert_settings(self, user_option):
         :param user_option: alert option key, typically 'mail:alert'
         :return: A dictionary in format {<user_id>: <int_alert_value>}
         """
-        from sentry.models import UserOption
+        from sentry.models import UserOption, User
+
+        member_alert_disabled = {}
```

Comment:
Why are we only returning disabled members now, rather than both enabled/disabled?

Reply:
This previously was returning the incorrect `value` since it was filtering by `key="mail:alert"`. Depending on what was selected in the Fine-Tuning section (like Default or On), we were using the `value` of that rather than the `value` of the Disabled/Enabled key (`key="subscribed_by_default"`).

```
DEFAULT DISABLED key='subscribe_by_default', value=0
FINE TUNING W/DEFAULT OR ON key='mail:alert', value=1

<UserOption at 0x11a70cc88: id=5, user_id=25, project_id=None, organization_id=None, key='workflow:notifications', value='1'>,
<UserOption at 0x11a70ccf8: id=1, user_id=25, project_id=None, organization_id=None, key='subscribe_by_default', value=0>,
<UserOption at 0x11a70cd68: id=19, user_id=25, project_id=2, organization_id=None, key='mail:alert', value=1>

DEFAULT ENABLED key='subscribe_by_default', value=1
FINE TUNING OFF key='mail:alert', value=0
FINE TUNING ON key='mail:alert', value=1

<UserOption at 0x11a70d128: id=13, user_id=26, project_id=None, organization_id=None, key='subscribe_by_default', value=1>,
<UserOption at 0x11a70d198: id=20, user_id=26, project_id=2, organization_id=None, key='mail:alert', value=0>
```

Reply:
Ahh got it, and I'll reach out to them to see what they're changing!